README.md

```diff
@@ -2,410 +2,17 @@
-| Rook              | [Enable multiple network interfaces for Rook storage providers](https://summerofcode.withgoogle.com/archive/2019/projects/5963949340622848/)                | Dmitry Yusupov                                     | Giovan Isa Musthofa |
-| TiKV              | [Proposal for Auto-tune RocksDB](https://summerofcode.withgoogle.com/archive/2019/projects/4737753357484032/)                                               | Wu                                                 | Yuanli Wang         |
-
-#### 2018
-
-| CNCF Projects | GSoC Project                                                                                                                                | Mentor(s)                                      | Mentee          |
-| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------- | --------------- |
-| containerd    | [Integrate Containerd with Katacontainers](https://summerofcode.withgoogle.com/archive/2018/projects/6403629539917824/)                     | Harry Zhang                                    | Jian Liu        |
-| CoreDNS       | [Conditional Name Server Identifier - CoreDNS](https://summerofcode.withgoogle.com/archive/2018/projects/4869116712189952/)                 | Miek Gieben, Yong Tang, John Belamaric         | Jiacheng Xu     |
-| Envoy         | [Extending Envoy's fuzzing coverage](https://summerofcode.withgoogle.com/archive/2018/projects/5597459212402688/)                           | Matt Klein, Constance Caramanolis, Harvey Tuch | Anirudh M       |
-| Kubernetes    | [Storage API for Aggregated API Servers](https://summerofcode.withgoogle.com/archive/2018/projects/5952269212188672/)                       | David Eads, Stefan Schimanski                  | Marko Mudrinić  |
-| Prometheus    | [Building a testing & benchmarking environment for Prometheus](https://summerofcode.withgoogle.com/archive/2018/projects/6728419798679552/) | Krasi Georgiev, gouthamve                      | Harsh_Agarwal   |
-| Prometheus    | [Composite Label Indices & Alerts rule testing](https://summerofcode.withgoogle.com/archive/2018/projects/4516877250330624/)                | gouthamve                                      | Ganesh Vernekar |
-| Rook          | [Add Network File System (NFS) as a Rook storage backend](https://summerofcode.withgoogle.com/archive/2018/projects/5713541642321920/)      | Travs Nielsen, jbw976                          | Rohan Gupta     |
+| [LFX Mentorship (ex-CommunityBridge)](https://mentorship.lfx.linuxfoundation.org) | Mentoring initiative by the Linux Foundation                                                                                              | [lfx-mentorship](https://github.com/cncf/mentoring/blob/main/programs/lfx-mentorship/README.md) |
```

Comment:
Nitpicking: use relative links so that the previews are also working?

When I check https://github.com/cncf/mentoring/tree/v2, I see links go to `main` branch.

There are other links in this PR I would suggest doing the same. 
I don't want to spam with the same suggestion :)

Reply:
Good point, I will look for those links and update

Reply:
@nate-double-u , updated via https://github.com/cncf/mentoring/pull/769